spec/controllers/idv/in_person_controller_spec.rb

```diff
@@ -82,7 +82,7 @@
               it 'redirects to the first step' do
                 get :index
 
-                expect(response).to redirect_to idv_in_person_step_url(step: :state_id)
+                expect(response).to redirect_to idv_in_person_proofing_state_id_path
               end
             end
```

Comment:
Default in specs is now enabled - looking at method `confirm_in_person_state_id_step_complete`in app/controllers/idv/in_person/address_controller.rb- this change looks good! You should be directed to the controller version of state id.  👏